lib/cdo/graphics/certificate_image.rb

```diff
@@ -134,7 +130,7 @@ def create_course_certificate_image(name, course=nil, sponsor=nil, course_title=
     sponsor = donor[:name_s]
   end
 
-  sponsor_message = I18n.t('certificate.sponsor_message', sponsor_name: sponsor)
+  sponsor_message = I18n.t('certificate_sponsor_message', sponsor_name: sponsor)
```

Comment:
nit; I'd love to see a comment calling out that this key exists in both pegasus and dashboard, just for future reference

Reply:
done